docs/content/quickstart.md

```diff
@@ -3,116 +3,127 @@ title: QuickStart Guide
+To see the bundle installations, use the `porter list` command. 
 
 ```
-porter install
+porter list
 ```
 
-If you wish to uninstall the bundle, you can use `porter uninstall`:
-
+Sample Output:
 ```
-porter uninstall
+NAME       CREATED          MODIFIED         LAST ACTION   LAST STATUS
+HELLO      21 minutes ago   21 minutes ago   install       succeeded
 ```
 
-## Publish the bundle
+In this example, it shows the bundle metadata along with the creation time, modification time, the last action that was performed, and the status of the last action. The default name of the bundle, HELLO, is used since there was no installation name specified at the command line. 
 
-When you are ready to share your bundle, the next step is publishing it to an
-OCI registry such as Docker Hub or Quay.
+## Show
 
-You must authenticate with `docker login` before publishing the bundle. Make
-sure that the `tag` listed in your `porter.yaml` is a reference to which the
-currently logged in user has write permission.
+To see information about an installation, use the `porter show` command with the name of the installation.
 
-```yaml
-tag: myregistry/porter-hello:v0.1.0
 ```
-
-Now run `porter publish` and porter will push the invocation image and bundle to
-the locations specified in the **porter.yaml** file:
-
+porter show HELLO
 ```
-porter publish
+
+Sample Output:
 ```
+Name: HELLO
+Created: 48 minutes ago
+Modified: 36 minutes ago
+
+History:
+----------------------------------------
+  Action     Timestamp       Status
+----------------------------------------
+  install    48 minutes ago  succeeded
+  uninstall  42 minutes ago  succeeded
+  install    36 minutes ago  succeeded
+  ```
 
-## Install from the registry
 
-Now that your bundle is in a registry, anyone can use a [CNAB-compliant
-tool][tools], not just Porter, to install the bundle. 
+## Run the upgrade instructions from the bundle
 
-Previously when we use
-`porter install` when we were in the same directory as a porter bundle, we
-didn't specify an installation name to create, so Porter defaulted the
-installation to the name of the bundle. This time we will explicitly name the
-installation "demo".
+To run the upgrade instructions from the bundle, use `porter upgrade`.
 
 ```
-porter install demo --tag getporter/porter-hello:v0.1.0
+porter upgrade 
 ```
 
-[tools]: https://cnab.io/community-projects/#tools
+Sample Output:
+```
+upgrading HELLO...
+executing upgrade action from HELLO (installation: HELLO)
+Upgrade Hello World
+Upgraded to World 2.0
+execution completed successfully!
+```
 
 ## Cleanup
 
+To clean up the resources installed from the bundle, use the `porter uninstall` command. 
+
+```
+porter uninstall HELLO
 ```
-porter uninstall demo
-```
\ No newline at end of file
+
+## Next Steps 
+
+So in this quickstart, you learned how to use some of the features of the porter cli to manage and inspect bundles. Next, learn more about using Porter with parameters.
```

Comment:
I'm going to avoid using "inspect" here because it's another command.

```suggestion
So in this quickstart, you learned how to use some of the features of the porter CLI to manage bundles. Next, learn more about using Porter with parameters.
```